port_ocean/core/handlers/entity_processor/jq_entity_processor.py

```diff
@@ -70,22 +70,31 @@ async def _search_as_object(
     async def _calculate_entities(
         self, mapping: ResourceConfig, raw_data: list[dict[str, Any]]
     ) -> list[Entity]:
-        async def calculate_raw(data: dict[str, Any]) -> dict[str, Any]:
+        async def calculate_raw(data: dict[str, Any]) -> list[dict[str, Any]]:
```

Comment:
pls note that rn you are also sending the `items_to_parse` to get parsed at line 83
but if ull apply my suggestion to pop this property one level out it should be fine